src/utils/local_file_mods.py

```diff
@@ -159,3 +160,87 @@ def local_file_write_feather(filepath, df):
     """
     df.to_feather(filepath)
     return True
+
+
+def local_delete_file(path: str):
+    """
+    Delete a file on the local file system.
+
+    Returns
+    -------
+    True for successfully completed operation. Else False.
+    """
+    try:
+        os.remove(path)
+        return True
+    except OSError:
+        return False
```

Comment:
preferrable to have a log here?
```suggestion:
except OSError as e:
    logging.error(f"error deleting file at {path}: {e}")
    return False
```